upstream-community-operators/eclipse-che/eclipse-che.7.0.0-beta-3.0.csv.yaml

```diff
@@ -0,0 +1,357 @@
+  maturity: stable
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  version: 7.0.0-beta-3.0
+  maintainers:
+    - email: eivantsov@redhat.com
+      name: Eugene Ivantsov
+  description: >
+    A collaborative Kubernetes-native development solution that delivers OpenShift workspaces and in-browser IDE for rapid cloud application development.
+    This operator installs PostgreSQL, Keycloak, and the Eclipse Che server, as well as configures all three services.
+
+    ## Pre-Reqs
+
+    In addition to a standard namespaced role, this operator may require a **ClusterRole** that allows the operator service account to:
+
+
+    * list, get, create, update, patch, watch and delete oauthclients at a cluster scope (the operator creates oAuthclient and OpenShift v3 identity provider in Keycloak to enable Login With OpenShift in Eclipse Che)
+
+
+    The operator service account will require extra privileges if you enable `auth.openShiftoAuth` which is false in CR template by default.
+    After the operator is installed, grant the **che-operator** service account such privileges:
+    When **auth.openShiftoAuth** is enabled:
+
+
+    ```
+
+    oc create clusterrole che-operator --resource=oauthclients --verb=get,create,delete,update,list,watch
+
+    oc create clusterrolebinding che-operator --clusterrole=che-operator --serviceaccount=${NAMESPACE}:che-operator
+
+    ```
+
+
+
+    `${NAMESPACE}` is an OpenShift project where you installed the operator.
+
+
+    ## How to Install
+
+    Press the **Install** button, choose the upgrade strategy, and wait for the **Installed** Operator status.
+
+    When the operator is installed, create a new CR of Kind CheCluster (click the **Create New** button).
+    The CR spec contains all defaults (see below).
+
+    You can start using Eclipse Che when the CR status is set to **Available**, and you see a URL to Eclipse Che.
+
+    ## Defaults
+
+    By default, the operator deploys Eclipse Che with:
+
+    * Bundled PostgreSQL and Keycloak
+
+    * Per-Workspace PVC strategy
+
+    * Auto-generated passwords
+
+    * HTTP mode (non-secure routes)
+
+    * Regular login (no login with OpenShift)
+
+    ## Installation Options
+
+    Eclipse Che operator installation options include:
+
+    * Connection to external database and Keycloak
+
+    * Configuration of default passwords and object names
+
+    * TLS mode
+
+    * PVC strategy (once shared PVC for all workspaces, PVC per workspace, or PVC per volume)
+
+    * Authentication options
+
+    ### External Database and Keycloak
+
+    To instruct the operator to skip deploying PostgreSQL and Keycloak and connect to an existing DB and Keycloak instead:
+
+    * set respective fields to `true` in a custom resource spec
+    * provide the operator with connection and authentication details:
```

Comment:
Hi,
here you probably miss the empty line, because it's not render as second item but as:
* set respective fields to `true` in a custom resource spec * provide the operator with connection and authentication details:

you can test it [here](https://operatorhub.io/preview) and also it will be look probably little better when you should tab little the code section above them

Reply:
@ssimk0 thank you. Fixed